index.html

```diff
@@ -108,13 +108,51 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*END TEMPLATE* Contributor card END ________  -->
 
-        
         <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
```

Comment:
Please a blank line between the template instructional comment line and the start of your card.